src/config/dynamic-theme-fixes.config

```diff
@@ -949,6 +949,14 @@ INVERT
 
 ================================
 
+brainly.pl
+
+INVERT
+[id^="TexFormula"]
```

Comment:
Where can I find this?

Reply:
https://brainly.pl/zadanie/18770149
https://brainly.pl/zadanie/18770085
https://brainly.pl/zadanie/18770321
https://brainly.pl/zadanie/18770210 (that's why "not" is there)
https://brainly.pl/zadanie/18770196 (again, that's why "not" is there)